doc/nrf/gs_ins_fork.rst

```diff
@@ -3,58 +3,83 @@
 Forking the |NCS| repositories
 ##############################
 
-If you want to change any of the code that is provided by the |NCS| and contribute this updated code back to the |NCS|, you should create your own forks of the |NCS| repositories and clone these forks instead of the base repositories.
+If you want to change any of the code that is provided by the |NCS| and contribute this updated code back to the |NCS|, you should create your own forks of the |NCS| repositories.
 
-To fork and clone the repositories, complete the following steps:
+To fork the repositories and update your cloned repositories to use your fork, complete the following steps:
```

Comment:
Maybe link to the section that instructs users how to use west to clone the repos?